APIMatic.Core.Test/Api/HttpGet/ApiCallGetDynamicError.cs

```diff
@@ -0,0 +1,363 @@
+                .With(req =>
+                {
+                    Assert.AreEqual(ContentType.JSON.GetValue(), req.Headers.Accept.ToString());
+                    return true;
+                })
+                .Respond(HttpStatusCode.NotImplemented, content);
+
+            var apiCall = CreateSimpleApiCall<ServerResponse>()
+                .RequestBuilder(requestBuilderAction => requestBuilderAction.Setup(HttpMethod.Get, url))
+                .ResponseHandler(responseHandlerAction => responseHandlerAction
+                    .ErrorCase("501", CreateErrorCase("Not Implemented error: {$response.body}.", (reason, context) => new Child2Exception(reason, context), true)))
+                .ExecuteAsync();
+
+            // Act and Assert
+            var exp = Assert.Throws<Child2Exception>(() => CoreHelper.RunTask(apiCall));
+            Assert.AreEqual("Not Implemented error: {\"body c2\":{\"field\":\"This is a field of body in child exception class\",\"integer\":15,\"bool\":true}}.", exp.Message);
+            Assert.Null(exp.ExceptionData);
+        }
+
+        [Test]
+        public void ApiCall_GetDynamicError_Local_501ResponseBodyJpointer()
+        {
+            //Arrange
+            var url = "/apicall/error/dynamic/local/501/responsebody/jpointer";
+            var expected = new Dictionary<string, object>()
+            {
+                { "body c2", new Dictionary<string, object>
+                    {
+                        { "field", "This is a field of body in child exception class" },
+                        { "integer", 15},
+                        { "bool", true}
+                    }
+                }
+            };
+
+            var content = JsonContent.Create(expected);
+            handlerMock.When(GetCompleteUrl(url))
+                .With(req =>
+                {
+                    Assert.AreEqual(ContentType.JSON.GetValue(), req.Headers.Accept.ToString());
+                    return true;
+                })
+                .Respond(HttpStatusCode.NotImplemented, content);
+
+            var apiCall = CreateSimpleApiCall<ServerResponse>()
+                .RequestBuilder(requestBuilderAction => requestBuilderAction.Setup(HttpMethod.Get, url))
+                .ResponseHandler(responseHandlerAction => responseHandlerAction
+                    .ErrorCase("501", CreateErrorCase("Not Implemented error: {$response.body#}, {$response.body#/body c2/integer}, {$response.body#/body c2/bool}, {$response.body#/body c2/na}, {$response.body#/body c2}.", (reason, context) => new Child2Exception(reason, context), true)))
+                .ExecuteAsync();
+
+            // Act and Assert
+            var exp = Assert.Throws<Child2Exception>(() => CoreHelper.RunTask(apiCall));
+            Assert.AreEqual("Not Implemented error: , 15, True, , {\r\n  \"field\": \"This is a field of body in child exception class\",\r\n  \"integer\": 15,\r\n  \"bool\": true\r\n}.", exp.Message);
```

Comment:
This test is failing on linux environment
<img width="554" alt="image" src="https://user-images.githubusercontent.com/26116671/215000915-4de24ae3-5418-45e0-8109-8d0d5f45fbcb.png">